crates/api/src/chronicle_graphql/mod.rs

```diff
@@ -413,9 +413,65 @@ where
     mutation: Mutation,
 }
 
+pub struct JwksUri {
+    uri: Url,
+}
+
+impl JwksUri {
+    pub fn new(uri: Url) -> Self {
+        Self { uri }
+    }
+}
+
+impl std::fmt::Debug for JwksUri {
+    fn fmt(&self, fmt: &mut std::fmt::Formatter<'_>) -> Result<(), std::fmt::Error> {
+        write!(
+            fmt,
+            r#"JwksUri {{ uri: Url {{ scheme: {}, cannot_be_a_base: {}, username: {}, password: ***SECRET***, host: {:?}, port: {:?}, path: {}, query: {:?}, fragment: {:?} }} }}"#,
```

Comment:
I wonder if these need more `{:?}`, seems weird to see the only somewhat the usual debug format but then many bare strings, especially when they're empty.

Reply:
It's by design as those items do not implement `Debug`, but implement `Display`.